Formula/graphpath.rb

```diff
@@ -1,8 +1,14 @@
 class Graphpath < Formula
   desc "Generates ASCII network diagram from the route table of a Unix/Linux"
   homepage "https://bsdrp.net/"
+<<<<<<< HEAD
```

Comment:
This doesn't seem right

Reply:
Do you mean the `homepage` URI or `<<<<<<< HEAD`? The latter has been removed in subsequent commits.

Reply:
I meant the merge conflict

Reply:
I was pushing up an update to the tarball destination when I encountered the merge conflict and resolved it. What about the merge conflict looks off?